os/fs/driver/mtd/mtd_partition.c

```diff
@@ -427,11 +427,8 @@ static int part_ioctl(FAR struct mtd_dev_s *dev, int cmd, unsigned long arg)
 
 	case MTDIOC_BULKERASE: {
 		/* Erase the entire partition */
-
+		fvdbg("priv: %p, startblock: %d, startsector: %x, nsectors: %d\n", priv, priv->firstblock, priv->firstblock / priv->blkpererase, priv->neraseblocks);
```

Comment:
This is not necessary, we already know geometry info.

Reply:
Okay, deleted.